# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gpu-alloc/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gpu-alloc/Cargo.toml b/third_party/rust/gpu-alloc/Cargo.toml
--- a/third_party/rust/gpu-alloc/Cargo.toml
+++ b/third_party/rust/gpu-alloc/Cargo.toml
@@ -8,17 +8,17 @@
 # If you believe there's an error in this file please file an
 # issue against the rust-lang/cargo repository. If you're
 # editing this file be aware that the upstream Cargo.toml
 # will likely look very different (and much more reasonable)
 
 [package]
 edition = "2018"
 name = "gpu-alloc"
-version = "0.4.7"
+version = "0.5.0"
 authors = ["Zakarum <zakarumych@ya.ru>"]
 description = "Implementation agnostic memory allocator for Vulkan like APIs"
 homepage = "https://github.com/zakarumych/gpu-alloc"
 documentation = "https://docs.rs/gpu-alloc-types"
 readme = "../README.md"
 keywords = ["gpu", "vulkan", "allocation", "no-std"]
 categories = ["graphics", "memory-management", "no-std", "game-development"]
 license = "MIT OR Apache-2.0"
@@ -38,11 +38,10 @@ default-features = false
 
 [dependencies.tracing]
 version = "0.1"
 features = ["attributes"]
 optional = true
 default-features = false
 
 [features]
-default = ["std", "freelist"]
-freelist = []
+default = ["std"]
 std = []